internal/mesh/internal/controllers/sidecarproxy/builder/destination_builder.go

```diff
@@ -261,6 +261,15 @@ func (b *Builder) buildDestination(
 	}
 
 	for _, details := range targets {
+		// NOTE: we only emit clusters for DIRECT targets here. The others will
+		// be folded into one or more aggregate clusters somehow.
+		if details.Type != pbmesh.BackendTargetDetailsType_BACKEND_TARGET_DETAILS_TYPE_DIRECT {
+			continue
+		}
+
+		connectTimeout := durationpb.New(5 * time.Second)
```

Comment:
Can this come from `details.DestinationConfig.ConnectTimeout`?

Reply:
yes, but i have another branch for that b/c it's more complicated